concordium-sdk/src/test/testresources/smartcontractschema/lib.rs

```diff
@@ -0,0 +1,242 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+//! # A Concordium V1 smart contract
+use concordium_std::*;
+use core::fmt::Debug;
+
+/// Your smart contract state.
+#[derive(Serialize, SchemaType)]
+pub struct State {
+    // Your state
+}
+
+/// Your smart contract errors.
+#[derive(Debug, PartialEq, Eq, Reject, Serial, SchemaType)]
+enum Error {
+    /// Failed parsing the parameter.
+    #[from(ParseError)]
+    ParseParams,
+    /// Your error
+    YourError,
```

Comment:
What is "YourError" ?

Reply:
Emil said smart contracts must have errors - it is indeed copy paste from the template.